### PR TITLE
add OrganizedStatisticalOutlierRemoval

### PR DESCRIPTION
**Pull request by knorth55**
`StatisticalOutlierRemoval` is useful to remove ghost point cloud, PCL1.8 (melodic) does not support `keep_organized` option.
This PR support `keep_organized` option for Melodic + PCL 1.8.

### Note:
`keep_organized` options is supported in above `PCL1.9`, but not supported in `PCL1.8`
Melodic uses `PCL1.8`, so I implement this PR.
https://github.com/PointCloudLibrary/pcl/pull/1663

### TODO:
- [x] add documentation
- [x] add sample

### Input of `Statistical Outlier Removal`
Screenshot from 2021-06-06 00-54-28

### Output of `Statistical Outlier Removal`
Screenshot from 2021-06-06 00-54-40

@tohirose

### CODE REVIEW

**Comment by knorth55** [comment]
@tohirose 

with this PR, we can get better object bounding box.

### `OrganizedStatisticalOutlierRemoval`:
Screenshot from 2021-06-06 01-23-02

### No `OrganizedStatisticalOutlierRemoval`:
Screenshot from 2021-06-06 01-21-19

**Comment by knorth55** [comment]
I update this PR to add sample and test launch.
I also add doc, too.
sor_doc

**Review by k-okada — APPROVED** [review]
LGTM

File: jsk_pcl_ros/scripts/install_sample_data.py [hunk 1]
```diff
@@ -139,6 +139,11 @@ def download_data(**kwargs):
         md5='dddd2c6d7738ac19536a032aed8f9967',
     )
 
+    download_data(
+        path='sample/data/baxter_realsense_l515.bag',
+        url='https://drive.google.com/uc?id=1RKZLA4J2_lzhCtvMTWxGQh4ykVVEeSAd',
+        md5='4dab24bd6e6a7fd5b9e86d053352cdfd',
+    )
```
Comment (k-okada) [inline jsk_pcl_ros/scripts/install_sample_data.py:146]: @knorth55 did you update the `baxter_realsense_l515.bag` file recently?  It seems the file is updated on 11/17 
https://drive.google.com/drive/u/1/search?q=baxter_realsense_l515.bag

Screenshot from 2022-11-24 17-05-52

Comment (knorth55) [reply jsk_pcl_ros/scripts/install_sample_data.py:146]: I'm sorry, I update the rosbag file for this PR.

Comment (knorth55) [reply jsk_pcl_ros/scripts/install_sample_data.py:146]: not for this PR, for this PR https://github.com/jsk-ros-pkg/jsk_recognition/pull/2656

Comment (knorth55) [reply jsk_pcl_ros/scripts/install_sample_data.py:146]: I made a PR to fix this issue https://github.com/jsk-ros-pkg/jsk_recognition/pull/2751

**Pull request merged**